lib/bulk_upload/bulk_upload.rb

```diff
@@ -2,10 +2,12 @@
 
 module BulkUpload
   module BulkUpload
+    include AddressHeaders
+
     def self.add_location_data(data, organisation)
       data.each do |row|
-        location = organisation.locations.new(address: row[0], postcode: row[1])
-        row.slice(2..).each do |ip_address|
+        location = organisation.locations.new(address: UploadForm.build_address(row), postcode: row[POSTCODE]&.strip)
+        row.slice(IP_ADDRESS..).reject(&:blank?).each do |ip_address|
```

Comment:
maybe you could `strip` the ip address as well, seeing how all the other fields are